Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -385,6 +385,10 @@
     <string name="main_mem1_size">MEM1 Size</string>
     <string name="main_mem2_size">MEM2 Size</string>
     <string name="gpu_options">GPU Options</string>
+    <string name="vi_frequency_enable">Override VBI Frequency</string>
+    <string name="vi_frequency_enable_description">Makes the game run at a different frame rate without impacting audio, making the emulation less demanding when lowered, or improving smoothness when increased. This may affect game speed, as it is often tied to the frame rate.</string>
```

Comment:
To match the other descriptions, this should say `games` rather than `the game`.

Reply:
The `without impacting audio` bit  seems weird to me. Why even mention audio if it's unaffected?